src/clusterfuzz/_internal/bot/fuzzers/libfuzzer.py

```diff
@@ -1079,6 +1100,8 @@ def run_single_testcase(self,
       result = LibFuzzerCommon.run_single_testcase(self, device_testcase_path,
                                                    timeout, additional_args)
       result.output = self._append_logcat_output_if_needed(result.output)
+      if environment.is_android_emulator():
```

Comment:
nit: for consistency with `_append_logcat_output_if_needed`, can we move the if into the function and call it `_prepend_trusty_stacktrace_if_needed`. 

alternatively, would it make sense to instead generalise  `_append_logcat_output_if_needed` to something like `_add_logcat_output_if_needed` to both do the prepend and append?